static/docs/user-guide/external-outputs.md

```diff
@@ -84,17 +83,20 @@ need to specify it explicitly.
 ### HDFS
 
 ```dvc
-    # Add HDFS remote that will be used as a cache location for your hdfs files
+    # Add HDFS remote to be used as cache location for HDFS files
     $ dvc remote add hdfscache hdfs://user@example.com/cache
 
-    # Tell dvc to use your 'hdfscache' remote as hdfs cache location
+    # Tell dvc to use the 'hdfscache' remote as HDFS cache location
     $ dvc config cache.hdfs hdfscache
 
-    # Run your stage with external HDFS output
+    # Run the stage with external HDFS output
     $ dvc run \
               -d data.txt \
               -o hdfs://user@example.com/home/shared/data.txt \
               hdfs fs -copyFromLocal \
                                 data.txt \
                                 hdfs://user@example.com/home/shared/data.txt
 ```
+
+Note that as long as there is a `hdfs://...` path for your data, DVC can handle
+it. So systems like Hadoop, Hive, and HBase are supported!
```

Comment:
In this file (static/docs/user-guide/external-outputs.md) I re-worded a few paragraphs, re-factored (to make sure the paragraphs are well aligned to 80 chars), and added the note above per https://discordapp.com/channels/485586884165107732/565759186693128202/571017836424593409